proseco/characteristics_guide.py

```diff
@@ -96,7 +96,19 @@
             "RegionFrac": .05,
             },
            "Imposter": {
-            "CentroidOffsetLim": 4.5,
+            "CentroidOffsetLim": 3.5,
+            },
+          {"Stage": 6,
+           "SigErrMultiplier": 0,
+           "ASPQ1Lim": 20,
+           "MagLimit": [5.6, 10.3],
```

Comment:
Setting to 5.6 is a hack that should allow stars up to 2 * 0.1 away to be selected (5.8 brightest).  Not sure where to put that in comments in the config file.

Reply:
Why doesn't the SigErrMultiplier apply here? 

 Putting the comment at this line in code seems fine.  I would also add a comment at the top of the stages referencing some TWiki page or section where you document all the notebooks and analysis that bears on these params.

Reply:
Regarding SigErrMultiplier, that's due to a bunch of compromises here when I thought we didn't want to let scope creep including the hack at:

https://github.com/sot/proseco/blob/ea4fa051702b0fe322b6abeeddd73ed97a2ca56b/proseco/guide.py#L209

But it might make more sense to get rid of half of those compromises and not care if SigErrMultiplier is 1 for all of the later stages (which can change the spoiler status).

Reply:
Ugh, though if we just set SigErrMultiplier to 1 without other changes, there's also the issue that we end up with significant catalog diffs due to it not selecting faint stars (for example a 10.25 mag star with mag_err 0.06 now could be more than 1 mag_err fainter than the 10.3 limit of the stage...).  

Or could just try to redo this so it makes sense.